### PR TITLE
Readcity

### PR DESCRIPTION
**Pull request by eunicehassan3**
I updated the main.py and added stuff to the readme. I'm wasn't sure if I should add any conditionals to the readCity() function so I just left it without any. Let me know if I should change anything.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -17,11 +17,18 @@
 formatted_response = json.dumps(response_data, indent=4)
 print(formatted_response)
 
+def readCity():
+  while True:
```
Comment (DylanAlfonso13) [inline main.py]: I'm not sure if you need this while loop, I think you can just read in the city without it

File: main.py [hunk 2]
```diff
@@ -17,11 +17,18 @@
...
+    city = input("Please enter the name of your city: ")
+    location = city
+    return location
+    break
+
```
Comment (DylanAlfonso13) [inline main.py]: Also, we need the location returned to be encoded to a url, this link shows a library that does that for us: https://www.urlencoder.io/python/

For example, "New York" needs to be returned as "New%20York"

File: main.py [hunk 3]
```diff
@@ -17,11 +17,18 @@
 formatted_response = json.dumps(response_data, indent=4)
 print(formatted_response)
 
+def readCity():
+  while True:
+    city = input("Please enter the name of your city: ")
+    location = city
```
Comment (DylanAlfonso13) [inline main.py]: So I would say at the top of the main file just import urllib.parse 
& maybe here you can say location = urllib.parse.quote(city)

Comment (DylanAlfonso13) [reply main.py]: The comment below this one explains why sorry lol its out of order

**Review by DylanAlfonso13 — CHANGES_REQUESTED** [review]
General idea looks good!

File: main.py (hunk 2 above)
Comment (eunicehassan3) [reply main.py]: fixed it

**Review by DylanAlfonso13 — APPROVED** [review]

**Pull request merged**